packages/services/src/linkResolver.service.ts

```diff
@@ -96,46 +101,51 @@ export interface GS1LinkResponse extends ILinkResponse {
 }
 
 export const createLinkResolver = async (arg: ICreateLinkResolver): Promise<string> => {
-  const { dlrAPIUrl, linkResolver, linkResponses, qualifierPath, responseLinkType = 'all' } = arg;
-  let registerQualifierPath = qualifierPath;
-  if (arg.queryString) {
-    registerQualifierPath = qualifierPath.includes('?')
-      ? `${qualifierPath}&${arg.queryString}`
-      : `${qualifierPath}?${arg.queryString}`;
-  }
-  const params: GS1LinkResolver[] = [constructLinkResolver(linkResolver, linkResponses, registerQualifierPath)];
+  const {
+    dlrAPIUrl,
+    namespace,
+    linkRegisterPath = '/api/resolver',
+    linkResolver,
+    linkResponses,
+    qualifierPath,
+    responseLinkType = 'all',
+  } = arg;
+
+  const params: LinkResolver = constructLinkResolver(namespace, linkResolver, linkResponses, qualifierPath);
   try {
     privateAPI.setBearerTokenAuthorizationHeaders(arg.dlrAPIKey || '');
-    await privateAPI.post<string>(`${dlrAPIUrl}/resolver`, params);
+    await privateAPI.post<string>(`${dlrAPIUrl}${linkRegisterPath}`, params);
 
     const path =
       responseLinkType === 'all'
         ? '?linkType=all'
         : qualifierPath.includes('?')
           ? `${qualifierPath}&linkType=${responseLinkType}`
           : `${qualifierPath}?linkType=${responseLinkType}`;
-    return `${dlrAPIUrl}/${linkResolver.identificationKeyType}/${linkResolver.identificationKey}${path}`;
+    return `${dlrAPIUrl}/${namespace}/${linkResolver.identificationKeyType}/${linkResolver.identificationKey}${path}`;
   } catch (error) {
     throw new Error('Error creating link resolver');
   }
 };
 
 export const constructLinkResolver = (
+  namespace: string,
   linkResolver: ILinkResolver,
   linkResponses: ILinkResponse[],
   qualifierPath: string,
 ) => {
-  const gs1LinkResolver: GS1LinkResolver = {
+  const LinkResolver: LinkResolver = {
```

Comment:
I think the `LinkResolver` variable should follow the camelCase naming convention.